src/index.html

```diff
@@ -4,9 +4,341 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo"><img src="images/logo.png" alt="logo"></a>
+
+      <nav class="header__nav">
+        <ul class="header__nav-ul">
+          <li class="header__nav-list">
+            <a class="header__nav-link is-active" href="#apple">Apple</a>
+          </li>
+
+          <li class="header__nav-list">
+            <a class="header__nav-link" href="#apple">Samsung</a>
+          </li>
+
+          <li class="header__nav-list">
+            <a class="header__nav-link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="header__nav-list">
+            <a
+              class="header__nav-link"
+              href="#laptopsAndComputers"
+              data-qa="nav-hover"
+            >Laptops & Computers</a>
+          </li>
+
+          <li class="header__nav-list">
+            <a class="header__nav-link" href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="header__nav-list">
+            <a class="header__nav-link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="header__nav-list">
+            <a class="header__nav-link" href="#photo">Photo</a>
+          </li>
+
+          <li class="header__nav-list">
+            <a class="header__nav-link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+
+    </header>
+
+    <main>
+
+      <div class="cards">
+        <div class="card cards__card" data-qa="card">
+          <img
+            class="card__img"
+            src="images/imac.jpeg"
+            alt="imac"
+          >
+
+          <h2 class="card__description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+          <div class="card__product-code">Product code: 195434</div>
+
+          <div class="card__review">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__reviews">Reviews: 5</div>
+          </div>
+
+          <div class="card__price">
+            <div class="card__price-key">Price:</div>
+            <div class="card__price-value">$2,199</div>
+          </div>
+
+          <a
+            class="card__buy"
+            href="#buy"
+            data-qa="card-hover"
+          >buy</a>
```

Comment:
Fix code style everywhere
```suggestion
            data-qa="card-hover"
          >
            buy
          </a>
```